app/pages/nonsipp/memberpensionpayments/RemovePensionPaymentsPage.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.memberpensionpayments
+
+import config.Refined.Max300
+import models.SchemeId.Srn
+import pages.Page
+
+case class RemovePensionPaymentsPage(srn: Srn, index: Max300) extends Page {
+
+  override def toString: String = "removePensionPayments"
```

Comment:
do we need this ? Are we using "removePensionPayments" anywhere such as in our userAnswer ?

Reply:
It seems we don't need this, just checking now after that will push the code. Thanks